lib/components/Provider/index.tsx

```diff
@@ -186,15 +191,17 @@ export const NotificationAPIProvider: React.FunctionComponent<
       setLoadingNotifications(true);
 
       try {
-        await fetchNotifications(
-          initial ? new Date().toISOString() : oldestLoadedRef.current,
-          initial ? config.initialLoadMaxCount : 1000
-        );
+        if (config.connectionMode === 'live') {
```

Comment:
similar to https://github.com/notificationapi-com/notificationapi-js-client-sdk/blob/bb982a9aa2234453c4d379ee312d242abbab231d/src/index.ts#L247